lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -1092,11 +1115,11 @@ class ConvertElementwiseOp : public ConversionPattern {
              AtenEqScalarOp, AtenLtScalarOp, AtenLeScalarOp, AtenWhereSelfOp,
              AtenCeilOp, AtenGtTensorOp, AtenGeTensorOp, AtenEqTensorOp,
              AtenLtTensorOp, AtenLeTensorOp, AtenSubScalarOp, AtenAddScalarOp,
-             AtenThresholdOp, AtenThresholdBackwardOp, AtenCloneOp, AtenSinOp,
-             AtenCosOp, AtenNeScalarOp, AtenNegOp,
-             AtenMaskedFillTensorOp, AtenLogicalOrOp, AtenLogicalAndOp,
-             AtenLogicalXorOp, AtenLogicalNotOp, AtenTriuOp, AtenBitwiseNotOp,
-             AtenRoundOp, AtenFillScalarOp, AtenFillTensorOp>(op))
```

Comment:
The CI is failing because of these changes. I think a conflict was not resolved correctly. There are ops missing now

Reply:
Hi @ramiro050 , I corrected the changes . Kindly check now.